generators/app/templates/prettierignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+coverage
+dist
```

Comment:
Unless this problem had been fixed, we need to have it ignore the package.json because npm reformats it when it installs so you get a bunch of changes to that file and i don't really care enough.

Reply:
Normally `npm` will keep the amount of spaces you set

Reply:
It's not the spaces. Those are fine. It's the fact that npm formats it with JSON.stringify which puts things like `"files": ["dist"],` on multiple lines when prettier would keep it as is.